console/views.py

```diff
@@ -1,8 +1,41 @@
 import logging
+import requests
 
+from flask import render_template
+from flask import request
 from structlog import wrap_logger
 
-import console.settings as settings
+from console import app
+from console import settings
 
 logging.basicConfig(level=settings.LOGGING_LEVEL, format=settings.LOGGING_FORMAT)
 logger = wrap_logger(logging.getLogger(__name__))
+
+
+def send_data(url, data):
+    r = requests.post(url, data)
+    if r.status_code == 404:
+        logger.error('Could not connect to ' + url, status_code=r.status_code)
+        raise Exception('404 Error')
+    else:
```

Comment:
What about if `decrypt` gives you a `500` error? Should probably check the positive case (`200`) first and then deal with other errors.